ciao-cli/main.go

```diff
@@ -1031,7 +1038,12 @@ func main() {
 		if *imageName == "" {
 			fatalf("Missing required -image-name parameter")
 		}
-		createTenantImage(*identityUser, *identityPassword, id, *imageName)
+		if *imageVisibility != "private" && *imageVisibility != "public" {
```

Comment:
I'd prefer to use our own for now. Once our initial PR on gophercloud is accepted we can add the Set() and String() method as a 2nd PR and move to ImageVisibility then.
I'd prefer not to add those 2 methods now as it may look awkward for the gophercloud reviewer.